_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About search telemetry
+
+You can use search telemetry to analyze performance for success or failed search requests in OpenSearch Dashboards. OpenSearch stores telemetry data in the `.kibana_1` index.
+
+Because there are thousands of concurrent search request from OpenSearch Dashboards, the large traffic causes significant load in an OpenSearch cluster.
+
+By default it is disabled, so you need to do some configuration changes to enable it.
+
+OpenSearch clusters perform better with search telemetry turned off.
+{: .tip }
+
+To learn more about using telemetry data with OpenSearch Dashboards, see [Trace Analytics OpenSearch Dashboards plugin](docs/2.0/observability-plugin/trace/ta-dashboards/).
```

Comment:
Hi @alicejw-aws, I believe you're conflating search telemetry (which is usage data that OpenSearch collects about successful and failed search requests to your cluster), and the Trace Analytics plugin, which is for application performance monitoring/visualizing OTEL data. You can remove this link, the two are not related.

Reply:
ok thanks @lizsnyder - yes i removed the reference to trace analytics, and found a link to the "data plugin" they were mentioning in the issue: 
https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/plugins/data